lib/src/over_react_test/common_component_util.dart

```diff
@@ -390,17 +390,26 @@ void testRequiredProps(BuilderOnlyUiFactory factory, dynamic childrenFactory(),
   var nullableProps = <String>[];
   var requiredProps = <String>[];
 
+  // Because Component2's propTypes do not stop the component from
+  // rendering with missing required props, an error may need to be caught
+  // on mount to keep the tests running.
+  void wrapInTryCatch(Function callback) {
+    try {
+      callback();
+    } catch (_){}
+  }
+
   setUp(() {
     var jacket = mount(factory()(childrenFactory()), autoTearDown: false);
     var consumedProps = (jacket.getDartInstance() as component_base.UiComponent).consumedProps;
     jacket.unmount();
 
     consumedProps.forEach((ConsumedProps consumedProps) {
       consumedProps.props.forEach((PropDescriptor prop) {
-        if (prop.isRequired) {
-          requiredProps.add(prop.key);
-        } else if (prop.isNullable) {
```

Comment:
Was this ever getting hit for `nullableRequiredProp`? lol

Reply:
nope in the description of the PR i mention that, its cause all isNullable's are also isRequired.
> The common_component_util.dart file has faulty logic that would never add any propDescriptors to the nullableProps list leading to the nullable props group/test (that is a foreach loop) to pass because it doesn't iterate over anything.